mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -85,6 +85,46 @@ CREATE TABLE mint_txs (
     FOREIGN KEY (mint_config_id) REFERENCES mint_configs(id)
 );
 
+-- Processed gnosis safe transactions
+CREATE TABLE gnosis_safe_txs (
+    eth_tx_hash VARCHAR NOT NULL UNIQUE PRIMARY KEY,
+    raw_tx_json TEXT NOT NULL
+);
+
+-- Deposits to the gnosis safe.
+CREATE TABLE gnosis_safe_deposits (
+    -- Diesel requires having a primary key and sqlite doesn't allow 64 bit primay keys, so even though
+    -- we would've wanted to use the block_index for that we can't.
+    -- Must be nullable for auto-increment: https://www.sqlite.org/autoinc.html
```

Comment:
well, technically it's diesel interprets sqlite's auto-increment primary keys as 32bit ints because they are defined as `INTEGER PRIMARY KEY`, and if you put `BIGINT PRIMARY KEY` then sqlite no longer applies the ROWID aliasing to them...

I'll remove all these comments and put one at the top about this annoyance